### DISCUSSION TITLE
COMPARE PAGE IS BUFFERING FOREVER IT WAS WORKING BEFORE WHEN I STARTED WITH 1 ONE CATEGORY ONLY

### DISCUSSION THREAD

**Discussion by vickypro1001** [bagisto/bagisto#7511] [Q&A]
COMPARE PAGE IS BUFFERING FOREVER IT WAS WORKING BEFORE WHEN I STARTED WITH 1 ONE CATEGORY ONLY.
PLEASE HELP SOMEONE
![image](https://user-images.githubusercontent.com/77957938/232320558-fb147877-66ce-4368-b724-958a26a6000e.png)

**Comment by Amitk-webkul** [comment]
Hi @vickypro1001, Could you please let us know which version of Bagisto you are using?

**Comment by devansh-webkul** [comment]
Inactive from last one year, closing the discussion.
